src/libra_py/dynamics/qtag/qtag_calc.py

```diff
@@ -204,58 +209,115 @@ def basis_diag(m, dt, H, S, b):
 
     return(c_new)
 
-
-
-def wf_print_1D(ntraj,qpas,c,fileobj):
-    """Writes a 1D single-surface wavefunction to a file specified by *fileobj*, computed using the internal function psi(qpas,c,x0) with basis parameters *qpas* and coefficients *c*.
+def wf_calc_nD(dyn_params, plt_params, prefix):
```

Comment:
based on the description of what this function is doing, it sounds like it may bettter be called "basis_initialize" or something like that

Reply:
This is a post-processing function to reconstruct the wavefunction on an arbitrary grid from the trajectory output (for plotting), not an initialization.